Sensus.Android/Sensus.Android.csproj

```diff
@@ -20,7 +20,7 @@
     <AndroidUseLatestPlatformSdk>false</AndroidUseLatestPlatformSdk>
     <ReleaseVersion>2.0.0</ReleaseVersion>
     <TargetFrameworkVersion>v6.0</TargetFrameworkVersion>
-    <AssemblyName>SensusAndroid</AssemblyName>
+    <AssemblyName>Sensus.Android</AssemblyName>

```

Comment:
I removed the period a while back because it was causing a deployment issue of some kind (can't recall exactly what). If this is cosmetic, please revert.